core/src/data/clientGeoJsonSource.h

```diff
@@ -23,7 +23,8 @@ class ClientGeoJsonSource : public DataSource {
 
 public:
 
-    ClientGeoJsonSource(const std::string& _name, const std::string& _url, int32_t _maxZoom = 18);
+    ClientGeoJsonSource(const std::string& _name, const std::string& _url,
+                        int32_t _minDisplayZoom = 0, int32_t _maxDisplayZoom = INT32_MAX, int32_t _maxZoom = 18);
```

Comment:
Or a default value of `-1` can be used to ignore any culling of tiles based on min or max display zooms.